masonry/src/widget/textbox.rs

```diff
@@ -220,7 +220,22 @@ impl Widget for Textbox {
         }
     }
 
-    fn on_access_event(&mut self, _ctx: &mut EventCtx, _event: &AccessEvent) {
+    fn on_access_event(&mut self, ctx: &mut EventCtx, event: &AccessEvent) {
+        if event.target == ctx.widget_id() {
```

Comment:
I don't think this check should be necessary any more? Is this because of bubbling?

Reply:
I copied that check from the implementation of `on_access_event` in `Button`.

Reply:
Having said that, if this event does bubble in Masonry, then I believe the check is still needed.

Reply:
Yeah. I think it isn't needed in this case specifically because TextBox has no child widgets, but in general it would be needed.